common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -373,13 +374,31 @@ private static MojoPipeline loadMojoPipelineFromFile() {
       throw new RuntimeException("Could not load mojo from file: " + mojoFile);
     }
     try {
-      MojoPipeline mojoPipeline = MojoPipelineService.loadPipeline(mojoFile);
+      MojoPipeline mojoPipeline = loadMojoPipelineFromFile(mojoFile);
       log.info("Mojo pipeline successfully loaded ({}).", mojoPipeline.getUuid());
       return mojoPipeline;
     } catch (IOException e) {
       throw new RuntimeException("Unable to load mojo from " + mojoFile, e);
     } catch (LicenseException e) {
       throw new RuntimeException("License file not found", e);
+    } catch (Exception e) {
+      throw new RuntimeException("Unexpected exception occurs", e);
     }
   }
+
+  private static MojoPipeline loadMojoPipelineFromFile(File mojoFile)
+      throws IOException, LicenseException {
+    try {
+      return MojoPipelineService.loadPipeline(mojoFile, buildPipelineConfig());
+    } catch (IllegalArgumentException e) {
+      log.debug("Disable Prediction interval because it is only supports regression model", e);
+      return MojoPipelineService.loadPipeline(mojoFile);
```

Comment:
QQ: Is there a way to check support without expecting an exception (and one of generic type `IllegalArgumentException`?

If so, I'd recommend that check as a precondition, assuming there isn't any significant overhead.  Else, the catch-fallback LFTM is there is no other approach.

---

nit: the log message is a bit unclear.

Reply:
i could change to 'Disable Prediction interval as it is not supported for the given model' , wdyt ?